requirements.txt

```diff
@@ -47,3 +47,5 @@ xlrd==1.2.0; python_version >= '3.8'
 XlsxWriter==1.1.2
 xlwt==1.3.*
 zeep==3.4.0
+xmlsig==0.1.5
+xades==0.2.1
```

Comment:
I did not find any Debian / Ubuntu package of those 2 libs.
I'm sorry to say that it's a no go for me.

Reply:
Thanks, I open this for testing only, later we remove this libraries, we are avaluating this with @jco-odoo 

cc: @vbe-odoo